README.md

```
@@ -1,102 +1,79 @@
-![Screenshot2](Add screenshot 2 here with proper name)
-*Add caption explaining what this shows*
-
-![Screenshot3](Add screenshot 3 here with proper name)
-*Add caption explaining what this shows*
-
-# Diagrams
-![Workflow](Add your workflow/architecture diagram here)
-*Add caption explaining your workflow*
-
-For Hardware:
+https://drive.google.com/file/d/1-JstYz-hpJV4V2L4v5P_Rsf5Gag_Md61/view?usp=drivesdk 
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)